examples/auth/auth_server/lib/src/endpoints/jwt_refresh_endpoint.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:serverpod_auth_idp_server/core.dart';
+
+/// By extending [RefreshJwtTokensEndpoint], the JWT token refresh endpoint
+/// is made available on the server and enables automatic token refresh on the client.
+class JwtRefreshEndpoint extends RefreshJwtTokensEndpoint {}
```

Comment:
Suggestion: Inverting the name here might confuse users since we already invert the name on the generated client. So users will see on the project:

- RefreshJwtTokensEndpoint 
- JwtRefreshEndpoint
- EndpointRefreshJwtTokens
- EndopointJwtRefresh

```suggestion
import 'package:serverpod_auth_idp_server/core.dart' as core;

/// By extending [RefreshJwtTokensEndpoint], the JWT token refresh endpoint
/// is made available on the server and enables automatic token refresh on the client.
class RefreshJwtTokensEndpoint extends core.RefreshJwtTokensEndpoint {}
```